### ISSUE TITLE
stop processing if there are no het variants

### ISSUE THREAD

**Issue by inti** [secastel/phaser#4]
Hi,
Some times a chromosome or contig may no have any het variant to be analysed. on these case it would be useful if phaser.py would stop and exit instead of carrying on.
this is an example
```
##################################################
              Welcome to phASER v0.2
  Author: Stephane Castel (scastel@nygenome.org)
##################################################

#1. Loading heterozygous variants into intervals...
     loading VCF into memory...
     parsing VCF...
          0 total heterozygous variants, 0 indels excluded, 0 blacklisted variants
     creating genomic intervals...
#2. Retrieving reads that overlap heterozygous sites...
     Reads are being written to disk, this will impact performance.
     file: mapping/samples/gsnap/BELA.sorted.bam
          minimum mapq: 30
          retrieved 0 reads
Traceback (most recent call last):
  File "/home/shared/app/phaser/phaser/phaser.py", line 1756, in <module>
    main();
  File "/home/shared/app/phaser/phaser/phaser.py", line 301, in main
    use_as_cutoff, as_cutoff = calculate_as_cutoff(reads_in);
  File "/home/shared/app/phaser/phaser/phaser.py", line 1706, in calculate_as_cutoff
    percentile_cutoff = numpy.percentile(alignment_scores,args.as_q_cutoff*100);
  File "/home/ipedroso/anaconda/envs/phaser/lib/python2.7/site-packages/numpy/lib/function_base.py", line 3268, in percentile
    interpolation=interpolation)
  File "/home/ipedroso/anaconda/envs/phaser/lib/python2.7/site-packages/numpy/lib/function_base.py", line 2997, in _ureduce
    r = func(a, **kwargs)
  File "/home/ipedroso/anaconda/envs/phaser/lib/python2.7/site-packages/numpy/lib/function_base.py", line 3385, in _percentile
    x1 = take(ap, indices_below, axis=axis) * weights_below
  File "/home/ipedroso/anaconda/envs/phaser/lib/python2.7/site-packages/numpy/core/fromnumeric.py", line 124, in take
    return take(indices, axis, out, mode)
IndexError: cannot do a non-empty take from an empty axes.
```

cheers,
inti

**Comment by secastel** [comment]
Hi there,
Yes, thanks for bringing this to our attention. Definitely agree with you that the script should gracefully exit in the case that there are no heterozygous variants to use for phasing. I will incorporate this into the next update.

Thanks,
Stephane

**Comment by inti** [comment]
Hi there, same thing in the case when there are no reads. It sounds strange but it could happen if variant calling phaser and being run with different mapq thresholds.
Cheers,
Inti

**Comment by secastel** [comment]
This should be fixed in the latest version (v0.3). phASER will gracefully exit in the case of either 0 variants or 0 reads. Thanks for bringing it to our attention.
